### DISCUSSION TITLE
Version 1.8.1 - Notarisation and minimum OS requirement bump

### DISCUSSION THREAD

**Discussion by joelekstrom** [joelekstrom/fastmate#59] [General]
* App is now signed, notarised and sandboxed with hardened runtime. This should make Gatekeeper happy and installation easier. #58 
* Fixes a bug where e-mail titles containing quotation marks (") would not trigger a notification

The minimum macOS version has been bumped to 10.15 (Catalina). This allows me to modernise the codebase using Combine together with a Swift rewrite, which will hopefully make it easier for others to contribute. (#56) Work is already underway on the master branch and part of the app has been migrated, but I will do it slowly and piece by piece to make sure not to break anything. After the migration the plan is refactor for testability and add a proper test suite, again to make contribution easier.

Thanks for using Fastmate!

<hr /><em>This discussion was created from the release <a href='https://github.com/joelekstrom/fastmate/releases/tag/v1.8.1'>Version 1.8.1 - Notarisation and minimum OS requirement bump</a>.</em>

**Comment by Halyard44** [comment]
Thank you for releasing fastmate and this release. I use it every day.

**Reply by mdbraber** [reply]
Agree with @Halyard44 - great to see work progressin on this @joelekstrom !
